pkg/infoutil/infoutil.go

```diff
@@ -151,6 +155,49 @@ func ServerSemVer(ctx context.Context, client *containerd.Client) (*semver.Versi
 	return sv, nil
 }
 
+func getBuildKitVersion() dockercompat.ComponentVersion {
+	buildctlBinary, err := buildkitutil.BuildctlBinary()
+	if err != nil {
+		logrus.Warnf("unable to determine buildkit version: %s", err.Error())
+		return dockercompat.ComponentVersion{Name: "buildkit"}
+	}
+
+	stdout, err := exec.Command(buildctlBinary, "--version").Output()
+	if err != nil {
+		logrus.Warnf("unable to determine buildkit version: %s", err.Error())
+		return dockercompat.ComponentVersion{Name: "buildkit"}
+	}
+
+	versionStr := strings.Fields(strings.TrimSpace(string(stdout)))
+
+	return dockercompat.ComponentVersion{
+		Name:    "buildkit",
```

Comment:
@ktock Should this be "buildkit" or "buildkitd"?

Reply:
Maybe we should add both `buildctl` and `buildkitd`

Reply:
> Maybe we should add both buildctl and buildkitd

SGTM. But I'm worried about how can we get  `buildkitd` version correctly. `buildkitd` on `PATH` may be different from the one `buildctl` accesses.

Maybe we can display only `buildctl` version as of now. In the future, I believe we can use `buildctl debug info` (available since https://github.com/moby/buildkit/pull/2725) to get buildkitd version once it's released.

Reply:
> Maybe we can display only buildctl version as of now.

SGTM

Reply:
Should I rename this component as buildctl for now?

Reply:
Yes

Reply:
Maybe we should put `buildctl` to the client part.

Reply:
Right now, there is no component in client part. Adding it would be easy. Should I go ahead and make the change?

Reply:
@AkihiroSuda WDYT

Reply:
> Maybe we should put buildctl to the client part.

SGTM